### DISCUSSION TITLE
Shortcut for toggling mute for the focused window

### DISCUSSION THREAD

**Discussion by Arxareon** [File-New-Project/EarTrumpet#1643] [Ideas]
It'd be awesome to have a keyboard shortcut option for quickly muting and unmuting (toggling mute) for the application the window of which is currently being focused.

**Comment by riverar** [comment]
Hi there, try out EarTrumpet Actions in the dev builds (https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller)!
